src/utils/theme/index.js

```diff
@@ -5,31 +5,49 @@ export const COLORS = {
   black30percent: 'rgba(0,0,0,0.3)',
   black80percent: 'rgba(0,0,0,0.8)',
   black43: '#434343',
-  blackgrey: '#828282',
   blue: '#2F80ED',
+  blueLoyal: '#023B60',
   blueOnboarding: '#4782D7',
+  blueTanzanite: '#11516F',
   blueSea: '#55C2FF',
-  bondi_blue: '#00ADB5',
+  blueZaffre: '#0E24B3',
+  brilliance: '#FCFCFC',
   bunting: '#11243D',
   concrete: '#F2F2F2',
   emperor: '#4F4F4F',
   french_pass: '#b6e4fd',
-  gray: '#6A6A6A',
-  gray1: '#C4C4C4',
-  gray6: '#F2F2F2',
-  gray4: '#4A4A4A',
-  gray5: '#E8EBED',
+  greenMantis: '#79B45D',
   greyseries: '#333333',
-  holytosca: '#00ADB5',
-  holytosca30percent: '#00ADB54D',
-  lightBlue: '#2C67BC',
-  lightgrey: '#F5F5F5',
   mine_shaft: '#333333',
   pattens_blue: '#ddf2fe',
+  platinum: '#E6E6E6',
   porcelain: '#ecf0f1',
-  red: '#FF2E63',
   silver: '#BDBDBD',
-  white: '#FFFFFF'
+  waterspout: '#9DEDF1',
+  white: '#FFFFFF',
+  whiteSmoke: '#F4F4F4',
+
+  // need to remove
+  holytosca: '#00ADB5',
+  holytosca30percent: '#00ADB54D',
+  lightBlue: '#2F80ED',
+  gray1: '#C4C4C4',
+  gray6: '#F2F2F2',
+  gray4: '#4A4A4A',
+  bondi_blue: '#00ADB5',
+
+  // new color
+  signed_primary: '#4782D7',
+  signed_secondary: '#6295DD',
+  anon_primary: '#107793',
+  anon_secondary: '#3A9BB6',
+  radical_red: '#FF3466',
+  light_silver: '#D2D4DB',
+  balance_gray: '#9B9FA9',
+  gray: '#E8EBED',
+  lightgrey: '#F5F6F7',
+  redalert: '#FF2E63',
+  blackgrey: '#828282'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `COLORS` object seem to be mostly additions and modifications of color values. However, there are comments indicating that some colors need to be removed (`holytosca`, `holytosca30percent`, etc.). If these colors are no longer in use, it would be best to remove them to avoid confusion and maintain cleanliness in the codebase.

```diff
-  // need to remove
-  holytosca: '#00ADB5',
-  holytosca30percent: '#00ADB54D',
```

Also, it's important to note that the color names should follow a consistent naming convention for better readability and maintainability. For instance, some colors are named after their actual color like `blue`, `redalert`, while others are named based on their usage like `signed_primary`, `anon_secondary`. It would be better to stick to one approach.



<!-- This is an auto-generated comment by OSS CodeRabbit -->